mysql/cloudmysql/cloudmysql.go

```diff
@@ -56,7 +56,7 @@ type Params struct {
 }
 
 // Open opens a Cloud SQL database.
-func Open(ctx context.Context, certSource proxy.CertSource, params *Params) (*sql.DB, error) {
+func Open(ctx context.Context, certSource proxy.CertSource, params *Params, traceOpts ocsql.TraceOption) (*sql.DB, error) {
```

Comment:
It's unfortunate that this argument list is getting kind of long, and that this is a breaking change. Maybe add to Params instead?

Reply:
Done.

Reply:
Done.

Reply:
Sorry, I misunderstood how these are supposed to be used. An example is here:

https://github.com/opencensus-integrations/ocsql/blob/f67f654fe0de226036abcecd6354c4b8205b1afb/driver.go#L81